go-controller/pkg/ovn/pods.go

```diff
@@ -109,38 +108,14 @@ func (oc *Controller) syncPodsRetriable(pods []interface{}) error {
 		if err != nil {
 			return fmt.Errorf("could not generate ops to delete stale ports from logical switch %s (%+v)", n.Name, err)
 		}
-
-		// have to set up the hybrid Overlay interface address here because all the pod IPs have been allocated above
-		if config.HybridOverlay.Enabled {
-			// check the annotation
-			nodeHybridOverlayDRIP := n.Annotations[hotypes.HybridOverlayDRIP]
-			var hybridOverlayDRIP []string
-			if len(nodeHybridOverlayDRIP) > 0 {
-				hybridOverlayDRIP = strings.Split(nodeHybridOverlayDRIP, ",")
-
-			}
-			AllocatedHybridOverlayDRIPes, err := oc.lsManager.AllocateHybridOverlay(n.Name, hybridOverlayDRIP)
-			if err != nil {
-				return fmt.Errorf("cannot allocate hybrid overlay interface addresses: %v", err)
-			}
-			var sliceHybridOverlayDRIP []string
-			for _, ip := range AllocatedHybridOverlayDRIPes {
-				sliceHybridOverlayDRIP = append(sliceHybridOverlayDRIP, ip.IP.String())
-			}
-			err = oc.kube.SetAnnotationsOnNode(n.Name, map[string]interface{}{hotypes.HybridOverlayDRIP: strings.Join(sliceHybridOverlayDRIP, ",")})
-			if err != nil {
-				return fmt.Errorf("cannot set hybrid annotations on node %s - %v", n.Name, err)
-
-			}
-
-		}
 	}
 
 	_, err = libovsdbops.TransactAndCheck(oc.nbClient, ops)
 	if err != nil {
 		return fmt.Errorf("could not remove stale logicalPorts from switches (%+v)", err)
 	}
 
+	atomic.StoreUint32(&oc.allInitialPodsProcessed, 1)
```

Comment:
@JacobTanenbaum could even move this up to right below the pod loop (before the GetNodes()) right?

Reply:
yep done